.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+# **what?**
+# Run tests for dbt-codegen against supported adapters
+
+# **why?**
+# To ensure that dbt-codegen works as expected with all supported adapters
+
+# **when?**
+# On every PR, and every push to main and when manually triggered
+
+name: Package Integration Tests
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+  run-tests:
+      uses: dbt-labs/dbt-package-testing/.github/workflows/run_tox.yml@main
```

dbt-labs has tagged a commit for this workflow to ensure we don't accidentally break anyone during development.
```suggestion
      uses: dbt-labs/dbt-package-testing/.github/workflows/run_tox.yml@v1
```